packages/app/api-common/src/apiSchemas.spec.ts

```diff
@@ -117,7 +117,7 @@ describe('apiSchemas', () => {
               "origin": "string",
               "code": "invalid_format",
               "format": "uuid",
-              "pattern": "/^([0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-8][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}|00000000-0000-0000-0000-000000000000)$/",
+              "pattern": "/^([0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-8][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}|00000000-0000-0000-0000-000000000000|ffffffff-ffff-ffff-ffff-ffffffffffff)$/",
```

Comment:
did this get broken after some package update?

Reply:
yes, I think it must have been zod update, not sure why it wasn't failing before